package.json

```diff
@@ -12,8 +12,8 @@
     "fix:prettier": "prettier --write \"{packages,__tests__,!dist}/**/*.{ts,tsx,js,json,md,yml}\"",
     "build": "pnpm -r --stream build",
     "build:clean": "lerna clean -y && pnpm install && lerna run build:clean --concurrency 1",
-    "test:ci": "jest --config=jest.json",
-    "test": "jest --verbose --config=jest.json --coverage=true --detectOpenHandles",
+    "test:ci": "jest --config=jest.json && jest --config=packages/siop-oid4vp/jest.json",
```

Comment:
Can't we combine this into one jest run?

Reply:
The siop/vp repo had different ts features turned on.
With the current tsconfig of the openid4vc repo, there are many errors.
That's why I created a separate jest.json file. Its only purpose is to use a different tsconfig.
I quickly checked if it was possible to configure it in one file but couldn't get it working.
I think the correct approach is to refactor the repo to align with the stricter typing used elsewhere in the repo at some point.

Reply:
Okay that makes sense, and probably good to first get it merged before changing too much about the code

Reply:
Yeah, I noticed 2 months ago we had been developing the SIOPv2 package with strict mode off etc. Will fix that soon as a first step before going into the bigger refactor